portfolio/src/main/webapp/script.js

```diff
@@ -65,18 +65,18 @@ function assginCommentsToUI() {
     }
 }
 
-function createNewElement({key, emoji, name, text, id, imageUrl}) {
+function createNewElement({key, name, text, id, sentimentScore, imageUrl}) {
     const newElement = document.createElement('div');
     newElement.setAttribute('class', 'media margin-10-px');
     newElement.setAttribute('id', id);
     newElement.innerHTML = `
-        <img class='margin-10-px' ${emoji == 1 ? ("src='icons/emoji-smile.svg'") : 
-            (emoji == 2 ? "src='icons/emoji-neutral.svg'" : "src='icons/emoji-frown.svg'")} 
+        <img class='margin-10-px' ${sentimentScore >= 0.45 ? ("src='icons/emoji-smile.svg'") : 
+            (sentimentScore <= -0.45 ? "src='icons/emoji-frown.svg'" : "src='icons/emoji-neutral.svg'")} 
```

Comment:
how about add a variable for the emoji name like:
```
const emoji = sentimentScore >= 0.45 ? 'emoji-smile' : (sentimentScore <= -0.45 ? 'emoji-frown' : 'emoji-neutral');
newElement.innerHTML = `<img class='margin-10-px' src='icons/${emoji}'.svg`;
```

Reply:
Will do